Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -645,6 +664,8 @@ BoardInit (
     if (GenericCfgData != NULL) {
       SetPayloadId (GenericCfgData->PayloadId);
     }
+
+    PcdSet32S (PcdPlatformCpuInitHook, (UINT32)(UINTN) PlatformCpuInit);
```

Comment:
Why a PCD instead of a library call?

Reply:
Using PCD here can avoid defining NULL instance library for platform that does not need hook.